### PR TITLE
Added get and set volume_db methods to the documentation

### PR DESCRIPTION
**Pull request by shweshiki**

### CODE REVIEW

File: doc/classes/AudioStreamPlayer.xml [hunk 1]
```diff
@@ -45,6 +45,22 @@
 				Stops the audio.
 			</description>
 		</method>
+		<method name="set_volume_db">
+			<return type="void">
+			</return>
+			<argument index="0" name="volume_db" type="float">
+			</argument>
+			<description>
+				Sets the volume of the audio in decibels.
```
Comment (mhilbrunner) [inline doc/classes/AudioStreamPlayer.xml]: I'd go with "Sets the audio volume in decibels."

File: doc/classes/AudioStreamPlayer.xml [hunk 2]
```diff
@@ -45,6 +45,22 @@
...
+			</description>
+		</method>
+		<method name="get_volume_db">
+			<return type="float">
+			</return>
+			<description>
+				Returns the volume of the audio in decibels.
```
Comment (mhilbrunner) [inline doc/classes/AudioStreamPlayer.xml]: Same here: audio volume instead of "volume of the audio".

**Review by mhilbrunner — COMMENTED** [review]
I'd replace "volume of the audio" with "audio volume".

**Comment by akien-mga** [comment]
Thanks, but those are already documented in `<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db">`.

We still need to better display the setter and getter related to a given member variable, but it's a feature that those don't appear under `<methods>`.

The XML is also automatically generated by the Godot binary, so you can't add new tags manually, they will be overridden next time `doctool` is ran.

**Pull request closed without merging**